plotIt_ZA/SignalFiles.yml

```diff
@@ -2,7 +2,7 @@
 
 'HToZATo2L2B_MH-500_MA-100_13TeV-madgraph_*_histos.root':
   type: signal
-  cross-section: 1000
+  cross-section: 1000 #it should be "scale" instead of "cross-section". This is a factor that goes to multiply the x-sec in the databse (1 fb for the signal)
```

Comment:
then why not renaming this to `scale` ? The option [exists](https://github.com/cp3-llbb/plotIt/wiki/List-of-configuration-options) and I think it should have the same effect. But this particular line of code would be less ambiguous ?

Reply:
Oh well. Fair enough :P